README.md

```diff
@@ -22,8 +22,10 @@ are contained in the 'res' directory.
 The following programs and/or libaries need to be installed before compiling
 and installing the RMA:
 
-Asterisk (>= version 13.0.0)
-libcurl
+- Asterisk (>= version 13.0.0)
+- libcurl
+- pjproject
+- res_rtp_asterisk
```

Comment:
I believe he is referring to the addition of 'res_rtp_asterisk'. That it is an Asterisk module. I'd probably avoid adding it here too as a dependency since it is, or should be, part of the Asterisk install.